bletl/parsing/blpro.py

```diff
@@ -48,6 +48,13 @@
 }
 """Maps 1-based well numbers in MF mode by **measurement counting order** (→→→ ↓ ←←← ↓ →→→ ...) to alphanumeric well IDs."""
 
+_WELL_NUMM_TO_ID = {
+    1 + (colnumber + rownumber * 8): f"{row}{column:02d}"
+    for rownumber, row in enumerate("ABCDEF")
+    for colnumber, column in enumerate(reversed(range(1, 9)) if rownumber % 2 else range(1, 9))
+}
+"""Maps 1-based well numbers in non-microfluidic mode by **measurement counting order** (→→→ ↓ ←←← ↓ →→→ ...) to alphanumeric well IDs."""
```

Comment:
For continuity purposes:

```suggestion
"""Maps 1-based well numbers in non-MF mode by **measurement counting order** (→→→ ↓ ←←← ↓ →→→ ...) to alphanumeric well IDs."""
```